scripts/fuzz_opt.py

```diff
@@ -828,8 +830,11 @@ def can_compare_to_self(self):
 
             def can_compare_to_others(self):
                 # If not legalized, the JS will fail immediately, so no point to
-                # compare to others.
-                return LEGALIZE and not NANS
+                # compare to others. Relaxed SIMD allows different behavior
+                # between VMs (in principle we could compare to other D8
+                # variants, though TODO).
```

Comment:
What kind of variants?

Reply:
We have V8 with liftoff, V8 with turboshaft, see line 839+

Reply:
Right, thanks!